app/frontend/src/theme.ts

```diff
@@ -1,37 +1,69 @@
-import { createMuiTheme, ThemeOptions } from "@material-ui/core";
+import { createMuiTheme, makeStyles, ThemeOptions } from "@material-ui/core";
 import type {} from "@material-ui/lab/themeAugmentation";
 
 const spacing = (factor: number) => `${0.5 * factor}rem`;
-const borderRadius = 10;
+const borderRadius = 12;
+
+const useGlobalStyles = makeStyles((theme) => ({
+  h1Large: {
```

Comment:
Is there any reason why we couldn't include this in the main theme too?

Reply:
Custom variants are in v5 but not v4

Reply:
This [stackoverflow](https://stackoverflow.com/questions/58168798/add-custom-theme-variable-object-to-createmuitheme) answer seem to suggest it works since v4.4.3 🤔  but I think you have to declare the additional types [like in here](https://material-ui.com/customization/theming/#custom-variables) for TypeScript to stop complaining.

Reply:
Maybe v5 has better generic types so this wouldn't be necessary once it's out of beta?

Reply:
You can but you can only access it from the theme object, not as a variant of typography. https://github.com/mui-org/material-ui/issues/22257